src/test/java/posters/pageobjects/pages/checkout/CartPage.java

```diff
@@ -36,9 +36,10 @@
 
     @Override
     @Step("ensure this is a cart page")
-    public void isExpectedPage()
+    public CartPage isExpectedPage()
     {
         cartTable.should(exist);
```

Comment:
Please call `super.isExpectedPage();` at first in case we would add some default functionality.

Reply:
@ufjena this hasn't been covered yet.